logback/src/main/java/com/linecorp/armeria/common/logback/LoggingEventWrapper.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.logback;
+
+import java.util.Map;
+
+import org.slf4j.Marker;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.IThrowableProxy;
+import ch.qos.logback.classic.spi.LoggerContextVO;
+
+final class LoggingEventWrapper implements ILoggingEvent {
```

Comment:
If Logback maintainers are not interested in https://github.com/qos-ch/logback/pull/614,
I'm OK to extend `LoggingEvent` with some TODO.
```
// TODO(ikhoon): Use `ILoggingEvent` instead of `LoggingEvent` once https://github.com/qos-ch/logback/pull/614 is merged.
```

Reply:
Nice!